code/controllers/subsystem/ships.dm

```diff
@@ -240,7 +247,15 @@ SUBSYSTEM_DEF(ship)
 				if(istype(S)) // fix for runtime (ship might have ceased to exist during the spawn)
 					broadcast_message("<span class=notice>Shot hit [faction2prefix(S)] shields. [faction2prefix(S)] ship shields at [S.shield_strength / initial(S.shield_strength) * 100]%!</span>",notice_sound,S)
 		return
+	if(attack_data.emp_attack && C.online) //If we do EMP damage, turn the component off
```

Comment:
we should make emp_attack a flag for attack_data. not necesary for this PR but is something to remember

Reply:
or is it a numeric value? cba to check :")

Reply:
~~disregard the statement or the flag bit?~~

Reply:
this entire comment